packages/icons/src/iacm-blue.svg

```diff
@@ -1 +1 @@
-<svg fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path clip-rule="evenodd" d="m21.1369 7.84538-2.7083 1.5076-5.6513-3.23172c-.2975-.17013-.6656-.17013-.9631 0l-5.43593 3.10852c-.63067.36065-.63068 1.25262 0 1.61332l5.58923 3.1962c.1742.0996.3763.1111.5521.0481.057-.0133.1132-.0347.1669-.0646l8.5595-4.76469-.0507 5.87549c-.011 1.2788-.6974 2.4564-1.8046 3.0958l-5.6926 3.2871c-1.1071.6393-2.4698.645-3.5822.0148l-5.53116-3.1332c-1.14106-.6463-1.84188-1.861-1.83058-3.1728l.02815-3.2677 9.13239 5.12c.0306.0171.0619.0315.0938.0431.2021.1413.4783.1623.7067.0317l6.0384-3.453c.3102-.1775.4148-.5674.2335-.8711-.1812-.3036-.5797-.4059-.8899-.2285l-5.7113 3.266-9.54358-5.3506c-.0156-.0087-.0314-.0167-.04737-.024l.014-1.62473c.01102-1.27887.69749-2.45647 1.80465-3.0958l5.6926-3.28716c1.1071-.63933 2.4698-.64497 3.5822-.01482l5.5312 3.1332c.8646.48981 1.4765 1.306 1.7173 2.24349zm-8.828 4.91422-4.76197-2.7232 4.74877-2.71556 4.8259 2.75966z" fill="#0278d5" fill-rule="evenodd"/></svg>
\ No newline at end of file
+<svg fill="none" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path clip-rule="evenodd" d="m5.00018 6.73536.18739 11.30944c.00875.473.33224.8764.81092 1.0111l12.37961 2.8965c.7924.223 1.6212-.3676 1.6154-1.1512l-.1317-17.73089c-.0056-.7569-.7906-1.2592-1.5667-1.00255l-12.43534 3.52497c-.51814.17135-.86892.63763-.85958 1.14263zm2.57541 4.06574c0-.2595.21036-.4699.46986-.4699h1.5502v-2.13873c0-.06584.04426-.12345.10788-.14041l3.74537-.99878v-1.3914c0-.2595.2104-.46987.4699-.46987h2.9367c.2595 0 .4699.21037.4699.46987v2.93669c0 .2595-.2104.46987-.4699.46987h-2.9367c-.2595 0-.4699-.21037-.4699-.46987v-1.41638l-3.71328.99022c-.00908.00242-.01541.01065-.01541.02006v2.13873h1.26189c.2595 0 .4699.2104.4699.4699v2.9367c0 .2595-.2104.4698-.4699.4698h-1.26189v2.08c0 .0094.00632.0177.01541.0201l3.71328.9902v-1.3577c0-.2595.2104-.4698.4699-.4698h2.9367c.2595 0 .4699.2103.4699.4698v2.9367c0 .2595-.2104.4699-.4699.4699h-2.9367c-.2595 0-.4699-.2104-.4699-.4699v-1.4501l-3.74537-.9988c-.06362-.0169-.10788-.0745-.10788-.1404v-2.08h-1.5502c-.2595 0-.46986-.2103-.46986-.4698zm.93973.4699v1.9969h1.99698v-1.9969zm5.87338-3.1423v-1.99695h1.9969v1.99695zm-.9398 2.6724c0-.2595.2104-.4699.4699-.4699h2.9367c.2595 0 .4699.2104.4699.4699v2.9367c0 .2595-.2104.4698-.4699.4698h-2.9367c-.2595 0-.4699-.2103-.4699-.4698zm.9398.4699v1.9969h1.9969v-1.9969zm0 7.1361v-1.997h1.9969v1.997z" fill="#0278d5" fill-rule="evenodd"/></svg>
```

Comment:
Can you remove the `width` and `height` attributes on these SVGs? Otherwise they won't scale

Reply:
Will do

Reply:
Done